src/components/molecules/TopBanner/TodayBalanceGameBanner/TodayBalanceGameBanner.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { PATH } from '@/constants/path';
+import { useNavigate } from 'react-router-dom';
+import { Check, CheckSmall } from '@/assets';
+import useIsMobile from '@/hooks/common/useIsMobile';
+import { GameContent } from '@/types/game';
+import * as S from './TodayBalanceGameBanner.style';
+
+interface TodayBalanceGameBannerProps {
+  index: number;
+  game: GameContent;
+}
+
+const TodayBalanceGameBanner = ({
+  index,
+  game,
+}: TodayBalanceGameBannerProps) => {
+  const navigate = useNavigate();
+  const isMobile = useIsMobile();
+  const onClickBanner = () => {
```

Comment:
이벤트 핸들러의 의미를 가지고 있다면, 
```suggestion
  const onBannerClick = () => {
```
이 괜찮아보이네요!